app/templates/SubmitJobAndWait.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.7"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.7 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.7/schedulerjob.xsd"
+    name="SubmitJobAndWait" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
+  <taskFlow>
+    <task name="SubmitJobAndWait">
+      <description>
+        <![CDATA[ The simplest task, ran by a groovy engine. ]]>
+      </description>
+      <variables>
+        <variable name="workflowURL" value="http://trydev.activeeon.com:8080/workflow-catalog/buckets/1/workflows/4?alt=xml" inherited="false" model="String"/>
+      </variables>
```

Comment:
trydev is an internal platform for testing purpose. URL to this platform should not appear in a release.
In this case, I would put:
http://try.activeeon.com:8080/workflow-catalog/buckets/1/workflows/4?alt=xml
something like:
http://repo.domain.com/workflow.xml
is another option.
But let's go for the first one.